assets/css/mform.css

```
@@ -593,4 +686,96 @@ section.repeater .repeater-group header + .mform > .mform-tabs .tab-content {
+        border-color: #3a4753;
+    }
+
+    body.rex-has-theme:not(.rex-theme-light) .mform-cl-multi-handle {
+        color: #5a6a78;
+    }
+
+    body.rex-has-theme:not(.rex-theme-light) .mform-cl-multi-remove {
+        color: #e57373 !important;
+    }
+}
 }
```

Stylelint (17.10.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)